omod/src/main/webapp/pages/taggedObjects.gsp

```diff
@@ -9,10 +9,10 @@
     ui.includeJavascript("uicommons", "services/personService.js")
 %>
 
-<div class="taggedObjects_display" ng-app="taggedObjects" ng-controller="taggedObjectsCtrl" ng-init="init()">
+<div class="taggedObjects_display" ng-app="taggedObjects" ng-controller="taggedObjectsCtrl" ng-init="loadTags()">
```

Comment:
In my previous comment I did mention you don't need to specify an init function, the function you pass to the controller is where you call from loadTags()